code/crates/test/src/block/transaction.rs

```diff
@@ -0,0 +1,174 @@
+use core::fmt;
+
+use bytes::Bytes;
+use malachitebft_proto::{self as proto};
+
+use crate::hash::Hash;
+
+/// Transaction
+#[derive(Clone, PartialEq, Eq, Ord, PartialOrd)]
+pub struct Transaction {
+    data: Bytes,
```

Comment:
Let's have the data be an enum with the Dummy tx as a variant, so that we can add more interesting tx types later down the road.